lib/sass/script/functions.rb

```diff
@@ -2053,15 +2055,23 @@ def list_separator(list)
     #   map-get(("foo": 1, "bar": 2), "baz") => null
     # @overload map_get($map, $key)
     #   @param $map [Sass::Script::Value::Map]
-    #   @param $key [Sass::Script::Value::Base]
-    # @return [Sass::Script::Value::Base] The value indexed by `$key`, or `null`
+    #   @param $keys [[Sass::Script::Value::Base]]
+    # @return [Sass::Script::Value::Base] The value indexed by `$keys`, or `null`
     #   if the map doesn't contain the given key
     # @raise [ArgumentError] if `$map` is not a map
-    def map_get(map, key)
+    def map_get(map, *keys)
       assert_type map, :Map, :map
-      map.to_h[key] || null
+      result = map
+      keys.each do |key|
+        if result.is_a?(Sass::Script::Value::Map) || result.is_a?(Hash)
+          result = result.to_h[key]
+        else
+          return null
+        end
+      end
+      result || null
     end
-    declare :map_get, [:map, :key]
+    declare :map_get, [:map, :key], :var_args => true
```

Comment:
The second parameter should be updated to be `$keys` rather than `$key`. However, you should also preserve the previous declaration, so that `map-get()` calls with named parameters continue to work.